### DISCUSSION TITLE
Live sync for all features does not work

### DISCUSSION THREAD

**Discussion by ghost** [dani-garcia/vaultwarden#3664] [Q&A]
### Subject of the issue

- live sync for  emergency contacts **does not work**.

Live synchronisation for emergency contacts doesn't work, either on the user side or on the admin side. When a new user is added, they have to either refresh the page, but this means they have to reconnect to the session; or they have to click on a new tab and return to the "emergency contact" tab.

### Deployment environment

- vaultwarden version: 1.28.1-alpine
- Install method: docker
- Clients used: all (tested: Browser, Firefox, Chrome)
- Reverse proxy and version: nginx proxy manager-v2.9.19
- Other relevant details: websocket works

### Steps to reproduce
- just add an account to the emergency contact

### Expected behaviour
- immediate synchronisation between the 2 user and admin sides when both sides accept until validation is complete

### Actual behaviour
- no live synchronisation, when a new user is added, they either have to refresh the page, but this means they have to reconnect to the session, or click on a new tab and return to the "emergency contact" tab.

BR Shydoo

**Comment by BlackDex** [comment]
For one there is no live sync for Emergency Contacts.
And if all others do not work, then i think you did not configured your web-sockets correctly.

**Reply by ghost** [reply]
However, websockets work between applications and extensions, so if I delete an element in the extension I immediately see it disappear between the different interfaces (Web, extension + Desktop).

EDIT : Why is there no synchronisation for emergency contacts ?
